package.json

```diff
@@ -80,6 +80,7 @@
     "@next/swc": "workspace:*",
     "@next/third-parties": "workspace:*",
     "@opentelemetry/api": "1.4.1",
+    "@picocss/pico": "1.5.10",
```

Comment:
Adding this as dev dep for testing e2e test case `app-dir/app-css` locally easier